### PR TITLE
Capitalize comments

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

File: packages/openapi-to-graphql-cli/.prettierignore [hunk 1]
```diff
@@ -1 +1,2 @@
-lib/
\ No newline at end of file
+lib/
+*.md
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql-cli/.prettierignore]: Some .md files contain code snippits, which Prettier is smart enough to recognize and will add semicolons...

File: packages/openapi-to-graphql/.prettierignore [hunk 2]
```diff
@@ -1 +1,2 @@
-lib/
\ No newline at end of file
+lib/
+*.md
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/.prettierignore]: Same thing.

**Comment by Alan-Cha** [comment]
Just trying to make the code base more consistent.

**Comment by ErikWittern** [comment]
Good work!

**Pull request merged**